inloop/tasks/models.py

```diff
@@ -18,6 +18,11 @@ class Meta:
 
     slug = models.SlugField(max_length=50, unique=True, help_text="URL name")
     name = models.CharField(unique=True, max_length=50, help_text="Category name")
+    priority = models.IntegerField(default=0, help_text="Priority order")
```

Comment:
Can you please change the field name to `display_order`. I don't think the term "priority" is a good choice here. Also, the help text should mention whether lower numbers will be displayed first (or the other way around?).